frontend/cypress/e2e/games/import/importOnlineGame.cy.ts

```diff
@@ -101,14 +105,23 @@ describe('Import Games Page - Import Online Games', () => {
         deleteCurrentGame();
     });
 
-    it('submits from Chess.com annotations URL', () => {
+    it('submits from Chess.com annotations URL (type A)', () => {
         importUrl(testUrls.chesscomAnalysisA);
+        // This particular analysis is missing headers
+        cy.getBySel('cancel-preflight').click();
         verifyGame({
             lastMove: 'Nxb6',
         });
         deleteCurrentGame();
     });
 
+    it('submits from Chess.com annotations URL (type B)', () => {
+        importUrl(testUrls.chesscomAnalysisB);
+        // This particular analysis is missing headers
+        verifyGame({});
```

Comment:
Shouldn't we still verify the last move? Also, it looks like the analysis does have headers? When I look at it on chesscom, white is named Test1 and black is named Test2. Is that not getting carried over during the import?

Reply:
My own copy pasta from Analysis A. I was stuck debugging the weird testing issues I mentioned that I never resolved - hadn't gotten to further refining the new test. I am going to look back into it.